packages/suite/src/utils/firmware/index.ts

```diff
@@ -28,6 +26,6 @@ export const getTextForStatus = (status: AppState['firmware']['status']) => {
         case 'check-fingerprint':
             return 'TR_CHECK_FINGERPRINT';
         default:
-            throw new Error('this state does not have human readable variant');
+            return null;
```

Comment:
Any reason why was the error throwing removed?

Reply:
Yeah, I don't wan't to show all of them in UI anymore - it was one of the issues of screens changing fast. started state only indicates that user initiated call to device but device has not sent any response yet.